pkgs/development/python-modules/analytics-python/default.nix

```
@@ -0,0 +1,52 @@
+{ lib
+, buildPythonPackage
+, fetchPypi
+, pythonOlder
+, requests
+, monotonic
+, backoff
+, python-dateutil
+}:
+
+buildPythonPackage rec {
+  pname = "analytics-python";
+  version = "1.4.0";
```

```suggestion
  version = "2.2.3";
```
I found that `1.4.0` is too old.
https://github.com/segmentio/analytics-python/tree/2.2.3